.travis.yml

```diff
@@ -173,12 +173,13 @@ before_script:
   - export LD_LIBRARY_PATH="$INSTALL_DIR/lib"
 
 script:
+  - find /local/mnt/workspace/ -name '*libclang_rt.asan-x86_64.so'
   - ninja
   - |
     if [[ "$FAST" == "ON" ]]; then
       ninja run_all
     elif [[ "$ASAN" = "ON" && "$TRAVIS_OS_NAME" == "linux" ]]; then
-      LD_PRELOAD=$(find /usr/ -name '*libclang_rt.asan-x86_64.so') ASAN_OPTIONS=symbolize=1 ASAN_SYMBOLIZER_PATH=$(which llvm-symbolizer) ninja run_all
+      LD_PRELOAD=$(find /local/mnt/workspace/tmp/ -name '*libclang_rt.asan-x86_64.so') ASAN_OPTIONS=symbolize=1 ASAN_SYMBOLIZER_PATH=$(which llvm-symbolizer) ninja run_all
```

Comment:
Simply use `find /` or does it take so long? @sanssecours what do you think?

Reply:
the thing is the asan library in there doesn't work it says linked to incompatible asan runtime there and points me to some strange path /local/mnt/workspace/tmp which i am now hackily investigating what it contains for us ;)

Reply:
ok apparently that path doesn't even exist and asan doesn't like the runtime in /usr/ although there are only 5.0.0 runtimes and i have no idea where else it magically picks it up from. lets just search / then.

Reply:
The search yielded nothing new, there seems to be only /usr/local/clang-5.0.0/lib/clang/5.0.0/lib/linux/libclang_rt.asan-x86_64.so but when i LD_PRELOAD on that it complains its  not the asan runtime linked against...

```
clang version 5.0.0 (tags/RELEASE_500/final)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /usr/local/clang-5.0.0/bin
```

this would also indicate that the runtime in local/clang/... should be correct so i'm not sure how to continue. I'm tempted to just exclude the process plugin in that constellation as we have our own asan build anyway. Its not executed yet though so lets see if that works as a replacement.

Reply:
Yes, it is difficult to find the correct one afterwards. I still suggest to do it via CMake, e.g., did you look at https://github.com/arsenm/sanitizers-cmake

Reply:
while this seems like a fine idea, in my experience changing such build-related things usually turn out to be a lot of work even if it only looks like changing a few simple line. So I'd prefer to follow another approach for this PR for now (like blacklisting it for this one travis asan build in case it works on our own asan build). We can open an issue about using such a cmake approach for the sanitizer for the future.